src/packets.cpp

```diff
@@ -142,6 +142,7 @@ void AOClient::pktSelectChar(AreaData* area, int argc, QStringList argv, AOPacke
 
     server->updateCharsTaken(area);
     sendPacket("PV", {"271828", "CID", argv[1]});
```

Comment:
Now that CIDs are implemented, it can't hurt to send the proper one in all of the PV packets.

Reply:
CID and user_id are different. CID refers to the position in the character list that the selected character is at.

Reply:
ohhh you mean replace 271828. yeah i gotcha